File: flow_controls/iooping/prime_number.py
number=int(input("Enter Number"))
flag=0
if number==1:
    print("NOT Prime")
else:
    for i in range(2,number):
        if (number%2==0):
            flag=flag+1
            break
        else:
            flag=0
    if flag==0:
        print("Prime")
    else:
        print("Not Prime")

File: javascript/primenumber.js
var number=17

var flag=0
//2,16
for(let i=2;i<number;i++){
if(number%i==0){
    flag+=1;
    break
}
else{
    flag=0}
}
if(flag==0){
    console.log("Prime");
}
else{
    console.log("Not Prime");
}